zerver/views/realm.py

```diff
@@ -276,6 +286,25 @@ def update_realm(
                 "move_messages_between_streams_limit_seconds"
             ] = move_messages_between_streams_limit_seconds
 
+    jitsi_server_url: Optional[str] = None
+    if jitsi_server_url_raw is not None:
+        jitsi_server_url = parse_jitsi_server_url(
+            jitsi_server_url_raw,
+            Realm.JITSI_SERVER_SPECIAL_VALUES_MAP,
+        )
+
+        # We handle "None" here separately, since do_set_realm_property is
+        # called only if setting value is not "None".
+        if jitsi_server_url is None and realm.jitsi_server_url is not None:
+            do_set_realm_property(
+                realm,
+                "jitsi_server_url",
+                jitsi_server_url,
+                acting_user=user_profile,
+            )
+
+            data["jitsi_server_url"] = jitsi_server_url
```

Comment:
This comment probably deserves some extra description or maybe an `else: pass` section -- I want it to be super clear that we expect `do_set_realm_property` to be called in the other cases, just via the loop below,